voltixApp/VoltixApp/Views/Send/SendCryptoDetailsView.swift

```diff
@@ -147,7 +147,10 @@ struct SendCryptoDetailsView: View {
     }
     
     private func getTitle(for text: String) -> some View {
-        Text(NSLocalizedString(text, comment: ""))
+        Text(
+            NSLocalizedString(text, comment: "")
+                .replacingOccurrences(of: "Fiat", with: CryptoPriceService.shared.defaultCurrency.uppercased())
```

Comment:
is it a search/replace error?

Reply:
Nope. I've changed the globalizations files as well, so we can replace the placeholder Fiat per (AUD, USD, ...)